app/src/components/AnimalCard.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const AnimalCard = ({ selectedIndividual, individualDetails }) => {
+  return (
+    <section>
+      {selectedIndividual ? (
```

Comment:
[fix] Move this logic to the parent component. This component should just be responsible for displaying the selected individual.